static/app/views/issueList/customViewsHeader.tsx

```diff
@@ -146,7 +152,7 @@ function CustomViewsIssueListHeaderTabsContent({
             orgSlug: organization.slug,
             groupSearchViews: newTabs.map(tab => ({
               // Do not send over an ID if it's a temporary id
-              ...(tab.id[0] !== '_' ? {id: tab.id} : {}),
+              ...(tab.id[0] !== '_' && !tab.id.startsWith('default') ? {id: tab.id} : {}),
```

Comment:
nit: would be nice to update the comment here to explain why we don't send default either